app/controllers/registrations_controller.rb

```diff
@@ -98,7 +96,20 @@ def short_label(label, truncate_limit = 60)
     label.truncate(truncate_limit, separator: /\s/)
   end
 
-  def registration_collection_ids_for_apo(apo_id)
-    Dor.find(apo_id).default_collections
+  def apo_id
+    druid = params[:apo_id]
+    raise ArgumentError if druid.nil?
+
+    return druid if druid.starts_with?('druid:')
+
+    "druid:#{druid}"
+  end
+
+  def object_client
+    Dor::Services::Client.object(apo_id)
+  end
+
+  def registration_collection_ids_for_apo
+    object_client.find.collectionsForRegistration
```

Comment:
Shouldn't this be in the administrative sub-schema?

Reply:
@jcoyne it should. and I'd expect at least one test to fail because of this. Looking into this now.

Reply:
@jcoyne OK, tweaked the tests, watched them fail, fixed them. Change now pushed.